application/models/gateway/Gammu_model.php

```diff
@@ -51,20 +51,20 @@ function _send_wap_link($data)
             $insert_id = $matches[1];
             if(empty($insert_id))
             {
-                die( "<div class=\"notif\"><font color='red'>Could Not Send Message. Make Sure Gammu Path is correctly set.</font></div>");
-                $f_ret = array('status' => 'Could Not Send Message. Make Sure Gammu Path is correctly set.'); //FIXME
+                die( '<div class="notif" style="color:red">'.lang('gammu_could_not_send_msg_gammu_path_incorrect').'</div>');
+                $f_ret = array('status' => lang('gammu_could_not_send_msg_gammu_path_incorrect')); //FIXME
             }
             else
             {
                 $this->db->update('outbox', array('SendingDateTime' => $data['date']), "ID = $insert_id" );
                 $this->Kalkun_model->add_sms_used($this->session->userdata('id_user'));
-                $f_ret = array('status' => 'Message queued');
+                $f_ret = array('status' => lang('gammu_msg_queued') );
```

Comment:
would suggest to make this generic and not gammu specific so that this can be reused